### PR TITLE
Updated apiVersion to be supported by kube 1.16 api 

### PR DESCRIPTION
**Pull request by jjvilleg**

### CODE REVIEW

**Review by evie-lau — CHANGES_REQUESTED** [review]
We'll also need the `spec.selector.matchLabel` field, and a 0.1.1 helm release

**Comment by jjvilleg** [comment]
Couldn't find original issue, so created a new one to track on the couchdb helm chart page. 
https://github.com/helm/charts/issues/17680

**Review by evie-lau — DISMISSED** [review]

**Comment by jjvilleg** [comment]
Ready to be merged @kevinwlau

File: chart/release/index.yaml [hunk 1]
```diff
@@ -2,7 +2,18 @@ apiVersion: v1
 entries:
   customer:
   - apiVersion: v1
-    created: "2019-09-05T13:51:06.32947-05:00"
+    created: 2019-10-03T13:14:16.525064-05:00
```
Comment (evie-lau) [inline chart/release/index.yaml]: Need to re-index the helm repo with the new 0.1.1 tar file, after the chart changes to use the new couchdb repo

**Review by evie-lau — CHANGES_REQUESTED** [review]

**Review by evie-lau — APPROVED** [review]

**Pull request merged**